command/agent/command.go

```diff
@@ -661,6 +662,12 @@ func (c *Command) setupTelemetry(config *Config) (*metrics.InmemSink, error) {
 		fanout = append(fanout, sink)
 	}
 
+	// Configure the prometheus sink
+	promSink, err := prometheus.NewPrometheusSink()
```

Comment:
This should be behind a config option: https://www.nomadproject.io/docs/agent/configuration/telemetry.html